pages/glossary/essential-text-presentation.md

```diff
@@ -0,0 +1,25 @@
+---
+title: Essential text presentation
+key: essential-text-presentation
+unambiguous: true
+objective: false
+---
+
+The presentation of text is considered essential in one of more of the following scenarios:
+
+1. The text and its presentation is part of a brand; or
+2. The text is part of a digitized image of a physical object; or
+3. The text is part of a free-form digital illustration; or
+3. The text is written, drawn, or otherwise hand-crafted; or
```

Comment:
```suggestion
4. The text is written, drawn, or otherwise hand-crafted; or
```

Reply:
I should have removed this one. It's a combination of 1 and 3.